deployments/engine/V3/partners/dev-example/deployment-config.dev.ts

```diff
@@ -0,0 +1,45 @@
+// This file is used to configure the deployment of the Engine Partner contracts
+// It is intended to be imported by the generic deployer in `/scripts/engine/V3/generic-v3-engine-deployer.ts`
+export const deployDetailsArray = [
+  {
+    network: "goerli",
+    // environment is only used for metadata purposes, and is not used in the deployment process
+    // Please set to "dev", "staging", or "mainnet", as appropriate
+    environment: "dev",
+    // if you want to use an existing admin ACL, set the address here (otherwise set as undefined)
```

Comment:
Nit: "otherwise set as undefined to deploy a new one"

Reply:
updated 👍